dev/preview/util/setup-access.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Will refactor this one next iteration as well, with the rest of those bash scripts. It's now used only in the werft job to set the correct contexts.